utils/crypto/bls/signer/rpcsigner/client.go

```diff
@@ -21,7 +24,24 @@ type Client struct {
 	pk     *bls.PublicKey
 }
 
-func NewClient(ctx context.Context, conn *grpc.ClientConn) (*Client, error) {
+func NewClient(ctx context.Context, rpcSignerURL string) (*Client, error) {
+	// TODO: figure out the best parameters here given the target block-time
+	opts := grpc.WithConnectParams(grpc.ConnectParams{
+		Backoff: backoff.DefaultConfig,
+	})
```

Comment:
Can we let the caller doesn't provide these options? Or was there a reason we chose to not pass this in as a parameter